### ISSUE TITLE
What is the DroidGuardHelper

### ISSUE THREAD

**Issue by Mannshoch** [microg/android_packages_apps_GmsCore#648]

**Comment by AndroidC-512L** [comment]
its needed to have safetynet with microg services

**Comment by Mannshoch** [comment]
does this mean that this app puts Google apps into some kind of virtual environment?

**Comment by ArchangeGabriel** [comment]
No. It only affects DroidGuard, the actual core of SafetyNet. It does nothing w.r.t. any Google app.

**Comment by Mannshoch** [comment]
Thanks for the Info. Maybe it would be a good Idea to made some description into the Wiki or  	README.md for the parts. So User know what which part really do.
